lua/gk.lua

```diff
@@ -11,8 +11,13 @@ function M.setup_block()
 	conf.lcore_end_id = 2
 	conf.flow_ht_size = 1024
 
+	local net_conf = gatekeeperc.get_net_conf()
```

Comment:
Make `net_conf` a parameter of `M.setup_block()` this will carry the need to have the network setup first.